frontend/web/components/pages/EnvironmentSettingsPage.js

```diff
@@ -327,6 +350,51 @@ const EnvironmentSettingsPage = class extends Component {
                             </Row>
                           )}
                         </div>
+                        {Utils.getFlagsmithHasFeature(
+                          'configure_hide_sensitive_data',
+                        ) && (
+                          <div>
+                            <Row space style={{ marginTop: '1.5rem' }}>
+                              <div className='col-md-8 pl-0'>
+                                <h3 className='m-b-0'>Hide sensitive data</h3>
+                                <p>
+                                  Exclude sensitive data from endpoints
+                                  returning flags and identity information to
+                                  the SDKs or via our REST API. For full
+                                  information on the excluded fields see
+                                  documentation{' '}
+                                  <ButtonLink
+                                    href='https://docs.flagsmith.com/advanced-use/hide-sensitive-data'
```

Comment:
Sorry, @gagantrivedi and I decided this should live with the rest of the administration documentation. 

```suggestion
                                    href='https://docs.flagsmith.com/advanced-use/system-administration#hide-sensitive-data'
```

Reply:
URL changed.